recipes/nb_conda_kernels/yum_requirements.txt

```diff
@@ -0,0 +1 @@
+fontconfig
```

Comment:
Sadly, no because PhantomJS seems to be using the system fontconfig. Maybe this can be fixed if there is a `phantomjs` package ( https://github.com/conda-forge/staged-recipes/pull/692 ).